Makefile

```diff
@@ -0,0 +1,11 @@
+NAME    := certmonger
+SRC_EXT := gz
+SOURCE  = https://releases.pagure.org/$(NAME)/$(NAME)-$(VERSION).tar.$(SRC_EXT)
+PATCHES = https://releases.pagure.org/$(NAME)/$(NAME)-$(VERSION).tar.$(SRC_EXT).sig
+PATCHES += 0006-NSS-crypto-policy-sets-minimum-RSA-and-DSA-key-size-.patch
```

Comment:
These shouldn't be needed with current [packaging](https://github.com/daos-stack/packaging/).